tests/foreman/ui/test_errata.py

```diff
@@ -316,7 +316,7 @@ def test_end_to_end(
 
     :parametrized: yes
 
-    :BZ: 2029192, 2265095
```

Comment:
BZ 2029192 was not closed as migrated, is this intended, and shouldn't we keep it as is?